website/sidebars.js

```diff
@@ -111,7 +111,12 @@ export default {
       label: "How-Tos",
       className: "category-howto",
       collapsible: false,
-      items: ["how-tos/how-to-sign-a-tx"],
+      items: [
+        "how-tos/how-to-sign-a-tx",
+        "how-tos/how-to-culculate-tx-hash",
+        "how-tos/how-to-culculate-script-hash",
+        "how-tos/how-to-culculate-code-hash",
```

Comment:
typo at ids. should be "calculate" instead of "culculate" (might need to change the id in each md file as well).